src/OpenColorIO/fileformats/FileFormatSpi1D.cpp

```diff
@@ -209,6 +233,26 @@ CachedFileRcPtr LocalFileFormat::read(std::istream & istream,
                     ThrowErrorMessage("Too many entries found", currentLine, "");
                 }
 
+                values.resize(components);
+
+                for (int i = 0; i < components; i++)
+                {
+                    float v = NAN;
+                    const auto result = NumberUtils::from_chars(inputLUT[i], inputLUT[i] + strlen(inputLUT[i]), v);
```

Comment:
[Question] Why using `strlen` here instead of 64 fixed size like is done elsewhere?